fonz/lookml.py

```diff
@@ -0,0 +1,122 @@
+from typing import List, Optional
+from fonz.exceptions import SqlError
+
+
+class LookMlObject:
+    def __repr__(self):
+        return f"{self.__class__.__name__}(name={self.name})"
+
+
+class Dimension(LookMlObject):
```

Comment:
Should the `Dimension` class be aware of what explore it is in or do we think that it will always be implemented as part of the `Explore` class?

Reply:
I thought about this and did some looking into how tree structures are implemented. It turns out that having a node that references both its children and its parents is not recommended (basically it's no longer a directed graph since you can traverse both directions). Let's leave it this way until we encounter a need that indicates otherwise.